pkg/client/cache/controller_test.go

```diff
@@ -284,6 +284,7 @@ func TestHammerController(t *testing.T) {
 	close(stop)
 
 	outputSetLock.Lock()
+	defer outputSetLock.Unlock()
```

Comment:
Sorry for the delay, I have been on paternity leave...

I suspect the original reason for omitting this is the TODO on line 282 above. I.e., it's purposely locking out any goroutines still running so they don't affect subsequent tests.